nix/postgres.nix

```diff
@@ -343,7 +343,7 @@ in
 
                 initdbArgs =
                   config.initdbArgs
-                  ++ (lib.optionals (config.superuser != null) ["-U" config.superuser]);
+                  ++ (lib.optionals (config.superuser != null) [ "-U" config.superuser ]);
```

Comment:
Wasn't this caught my nixci (the treefmt-nix check)?

Reply:
I don't think it did, or maybe it went unnoticed